recipes/quickfix/all/conanfile.py

```diff
@@ -11,8 +13,17 @@ class QuickfixConan(ConanFile):
     topics = ("conan", "QuickFIX", "FIX", "Financial Information Exchange", "libraries", "cpp")
     settings = "os", "compiler", "build_type", "arch"
     options = {"fPIC": [True, False],
-               "ssl":  [True, False]}
-    default_options = {"ssl": False, "fPIC": True}
+               "shared": [True, False],
+               "with_ssl":  [True, False],
```

Comment:
Usually I don't vote for changing options names. Once available, they can break the users, but you are the author of this recipe, and it's recent, I don' see any problem.

Reply:
I guess this update is already applied. You mean changing the option name `ssl` to `with_ssl`, right?

Reply:
Ok, I got it know. You're commenting on my choice for renaming the option 😳 
Closing this now.